packages/cosmic-proto/scripts/codegen.cjs

```diff
@@ -0,0 +1,101 @@
+const { join } = require('path');
+const process = require('process');
+const telescope = require('@cosmology/telescope').default;
+const rimraf = require('rimraf').rimrafSync;
+
+const protoDirs = [join(__dirname, '/../proto')];
+const outPath = join(__dirname, '../src/codegen');
```

Comment:
I'd prefer this, since `"codegen"` is just noise.
```suggestion
const outPath = join(__dirname, '../src');
```

Reply:
Isn't it helpful to distinguish which source files were generated by machine vs by human? @dckc asked for this explicitly in review